src/tests/conformance/post_handshake/broadcast/proposalpayload.rs

```diff
@@ -11,25 +18,23 @@ async fn c007_PROPOSAL_PAYLOAD_expect_after_connect() {
     // ZG-CONFORMANCE-007
 
     // Spin up a node instance.
-    let target = TempDir::new().expect("couldn't create a temporary directory");
-    let mut node = Node::builder()
-        .build(target.path())
-        .expect("unable to build the node");
+    let target = TempDir::new().expect(ERR_TEMPDIR_NEW);
+    let mut node = Node::builder().build(target.path()).expect(ERR_NODE_BUILD);
     node.start().await;
 
     // Create a synthetic node and enable handshaking.
     let mut synthetic_node = SyntheticNodeBuilder::default()
         .build()
         .await
-        .expect("unable to build a synthetic node");
+        .expect(ERR_SYNTH_BUILD);
 
-    let net_addr = node.net_addr().expect("network address not found");
+    let net_addr = node.net_addr().expect(ERR_NODE_ADDR);
 
     // Connect to the node and initiate the handshake.
     synthetic_node
         .connect(net_addr)
         .await
-        .expect("unable to connect");
+        .expect(ERR_NODE_CONNECT);
```

Comment:
I just noticed:
```suggestion
        .expect(ERR_SYNTH_CONNECT);
```
If we follow the rule: `ERR_<WHO>_<FUNCTION_NAME>`
`Who` is here our synthetic_node, not the node.

Reply:
Done.